packages/gatsby/src/bootstrap/load-plugins/validate.ts

```diff
@@ -280,7 +282,13 @@ export async function validateConfigPluginsOptions(
   config.plugins = plugins
 
   if (errors > 0) {
-    process.exit(1)
+    if (process.env.SKIP_PLUGIN_OPTION_VALIDATION) {
+      reporter.warn(`you have enabled the SKIP_PLUGIN_OPTION_VALIDATION flag,`)
```

Comment:
Nitpick: I'd rather we mention explicitly everytime that there is incorrect configuration, something like:

```suggestion
      reporter.warn(`your gatsby-config.js contains invalid plugin options`)
      reporter.warn(`because you have enabled the SKIP_PLUGIN_OPTION_VALIDATION flag, develop will try continuing`)
      reporter.warn(`but this could have unintended side effects and break your site`)
```